CODE_OF_CONDUCT.md

```diff
@@ -40,6 +40,14 @@ Project maintainers are responsible for clarifying the standards of
 acceptable behavior and are expected to take appropriate and fair
 corrective action in response to any instances of unacceptable behavior.
 
+Members of the official team are required to disclose any interests 
```

Comment:
Good catch, ill give this some thought.

Reply:
After thinking this through for a bit, i think the following method is the best:  the conflict of interest may posted in any of the community spaces, whether it is the forum or riot chat. if wished for, it may either be in a team only space, or a public one.
If it is determined to be a conflict indeed, then it will be made public. if it is determined that it is not a conflict, then it may be kept private(or made public if the member itself wishes to do so). This should give us a good balance between transparency and the privacy of team members: actual conflicts will be reported,  and in cases where it does not matter (so if its not a conflict) they may choose to keep it private.

I would love to get feedback on this, as its a quite important decision. @lizmcintyre @davegson

Reply:
> If it is determined to be a conflict indeed, then it will be made public. if it is determined that it is not a conflict, then it may be kept private(or made public if the member itself wishes to do so).

**It is critical that any believed conflict of interest be posted publicly for the sake of transparency.** The recent case where a whistleblower (Mikaela) identified a post as having a COI is the perfect example. Had she not come forward, we might never have known about the COI.

What's more, the person with the COI removed the COI label, and there are some who contended (and may still contend) that a conflict of interest never existed. (There was a COI from the moment Startpage offered to discuss an opportunity with a Team Member. Auditor here.)

I believe we also need a whistleblower policy to protect Team Members when they provide critical information to the public, like the recent COI.

Reply:
hey there liz!

Okay, so after thinking about this more, I agree with your first conclusion.

My idea now is that someone should firstdisclose his potentional conflict of interest. then the team should overlook whether something realy is a COI. if it is NOT a COI, it shall be mae public with the note that there was a discussion and with all the details, but that the member will keep his voting rights(as there is no COI). 

If there IS a COI however, it shall be made public, and the voting rights will be removed from the member on that topic. After that participation in discussions is allowed (in the form of opinions), but only if the person agrees to disclose his affiliation with every message, so people new to the discussion know whats up.

About a whistleblower policy, I would have to read a bit into that, as its pretty hard to implement something like that were it would be actually meaningfull. ill come back to you about that one in  few days. I do think it would be a good idea though, especially as our organization grows and more folks get involved.

What would you think of that?

Reply:
P.s. i already created a draft issue for a whistleblowers policy, so further discussion about it should be done on the appropriate issue:https://github.com/privacytoolsIO/.github/pull/5

Reply:
> After thinking this through for a bit, i think the following method is the best: the conflict of interest may posted in any of the community spaces, whether it is the forum or riot chat.

I think it has to be something that is not a real time communication platform first. The RTCs are too busy and active at all times and finding information and what has been said before is difficult in them.

The wording in the version of the CoI policy I previously saw seemed to call for a single place where to see all the CoIs, so maybe the team page should be extended or bios could get a new field for affiliations? That may again go to https://github.com/privacytoolsIO/.github/pull/3#discussion_r374658409 though or the question what is a significant affiliation?

Reply:
Above I was talking about the discustions about whether something is a COI or not, which may be in chat but could also be on the forum. The place where the reports/conclusions end up and are made visable is another question.